scripts/cli-test.sh

```diff
@@ -54,26 +54,39 @@ mine_blocks 10
 RECEIVED=$($FM_BTC_CLIENT getreceivedbyaddress $PEG_OUT_ADDR)
 [[ "$RECEIVED" = "0.00000500" ]]
 
-# outgoing lightning
-INVOICE="$($FM_LN2 invoice 100000 test test 1m | jq -e -r '.bolt11')"
-await_cln_block_processing
+# lightning tests
+await_lightning_node_block_processing
+
+# outgoing lightning payment via gateway
+ADD_INVOICE="$($FM_LNCLI addinvoice --amt_msat 100000)"
+INVOICE="$(echo $ADD_INVOICE| jq -e -r '.payment_request')"
+PAYMENT_HASH="$(echo $ADD_INVOICE| jq -e -r '.r_hash')"
 $FM_MINT_CLIENT ln-pay $INVOICE
 # Check that ln-gateway has received the ecash notes from the user payment
 # 100,000 sats + 100 sats without processing fee
 LN_GATEWAY_BALANCE="$($FM_GATEWAY_CLI balance $FED_ID | jq -e -r '.balance_msat')"
 [[ "$LN_GATEWAY_BALANCE" = "100100000" ]]
-INVOICE_RESULT="$($FM_LN2 waitinvoice test)"
-INVOICE_STATUS="$(echo $INVOICE_RESULT | jq -e -r '.status')"
-[[ "$INVOICE_STATUS" = "paid" ]]
+INVOICE_STATUS="$($FM_LNCLI lookupinvoice $PAYMENT_HASH | jq -e -r '.state')"
```

Comment:
Using lookup instead of waiting for the invoice works because we wait for the payment to succeed previously (i.e. the preimage reached the payer). This means the LN node *should* recognize the invoice as paid, since it gave away the preimage.

Reply:
Correct. Once `fedimint-cli ln-pay <invoice>` completes, that means that the LND node received the payment, so it should have marked that invoice as paid.